src/components/widgets/CheckboxesWidget.js

```diff
@@ -29,6 +30,7 @@ function CheckboxesWidget(props) {
             <input
               type="checkbox"
               id={`${id}_${index}`}
+              name={idToPath(`${id}.${index}`)}
```

Comment:
Did you mean to put an underscore here?

```suggestion
              name={idToPath(`${id}_${index}`)}
```

Reply:
The '.' is the correct format for the path in order to use lodash's pick() method.

Reply:
Right -- but `idToPath` takes in a path with underscores in it, then outputs a path with "."

Reply:
Ohh I see what you are saying. Ya I can change that for consistency, however it would get converted right back to a '.'

Reply:
Yes, please do!